src/web/core/src/main/java/org/geoserver/web/system/status/ConsoleInfoUtils.java

```diff
@@ -0,0 +1,70 @@
+/* (c) 2022 Open Source Geospatial Foundation - all rights reserved
+ * This code is licensed under the GPL 2.0 license, available at the root
+ * application directory.
+ */
+package org.geoserver.web.system.status;
+
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStreamReader;
+import java.lang.management.ManagementFactory;
+import java.lang.management.ThreadInfo;
+import java.util.Arrays;
+import java.util.Optional;
+import java.util.stream.Collectors;
+
+/** utility interface for thread and memory heap info */
+public interface ConsoleInfoUtils {
```

Comment:
Using an interface for utility methods is technically possible but quite weird (e.g. https://stackoverflow.com/questions/25098937/java-8-interface-with-static-methods-instead-of-static-util-class  or https://stackoverflow.com/questions/30905236/are-interfaces-a-valid-substitute-for-utility-classes-in-java-8).

I'd be more confortable with the classic final class with private constructor, and static imports in the place of usage.